### DISCUSSION TITLE
(12 > 3) more drives are expected to heal than parity, returned errors

### DISCUSSION THREAD

**Discussion by staybox** [minio/minio#18455] [Q&A] [answered]
Hello,

Error: (12 > 3) more drives are expected to heal than parity, returned errors: [<nil> <nil> <nil> <nil> <nil> <nil> <nil> <nil> <nil> <nil> <nil> <nil>] (dataErrs [file not found file not found file not found file not found file not found file not found file not found file not found file not found file not found file not found file not found]) -> earth-services-v2/forest/order-files/1228/composites/770390/770392/synthesis.tif(73f5f9fd-fbea-49b7-b165-beb523f67e85) (*errors.errorString)

How to fix it?

**Comment by klauspost** [comment, marked as answer]
The data is not there. Without knowing more there is nothing you can do.

My best guess is a partially completed delete operation.
